### ISSUE TITLE
Reproducing the results of pretrained models

### ISSUE THREAD

**Issue by samgregoost** [facebookresearch/meru#6]
Hi, thanks for the amazing work and code.

I've got a question about the reproducibility of the results provided by pre-trained models. Specifically, when zero-shot evaluation is done by the provided pretrained meru-small model, I can get around 36% accuracy on imagenet. 

However, when trained from scratch, using the default configs, the zero shot accuracty over imagenet is only around 27%. May I ask any possible reason for this discrepancy? Is it possible that the provided pre-trained models are initialized from the CLIP weights, and then fine tuned on redcaps?

Any insight would be appreciated.
